tests/Service/LoggingServiceFactoryTest.php

```diff
@@ -47,7 +47,7 @@ public function testFactoryCallingNonExistantFunctionThrowsError()
         $service = new VersionsService(
             $this->createMock('BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\VersionRepository'),
             $this->createMock('BBC\ProgrammesPagesService\Mapper\ProgrammesDbToDomain\VersionMapper'),
-            new NullAdapter()
+            $this->createMock('BBC\ProgrammesPagesService\Cache\CacheInterface')
```

Comment:
Agree in principle, we should probably do this in bulk rather than one at a time though.